### ISSUE TITLE
Make the FTP extension free

### ISSUE THREAD

**Issue by willmot** [humanmade/backupwordpress#903]
Rational

- It gives people an offsite backup option without them having to spend money
- It allows people to test an extension
- It should lead to more people purchasing the other extensions

**Comment by owaincuvelier** [comment]
Will it bundle with BackUpWordPress Free plugin?

**Comment by willmot** [comment]
I'd say so initially, although once we have #525 it might be fun to have it use that.

**Comment by owaincuvelier** [comment]
Yeah I like that idea too. When people going to try the free FTP we expose our other premium extensions ect. +!

**Comment by katmoody** [comment]
Adding a point in here - this would be an optimal offer available for optin to the mailing list, at least at some point?

**Comment by willmot** [comment]
That's a pretty good idea actually 👍
